php-cs-fixer

```diff
@@ -41,14 +41,22 @@ set_error_handler(function ($severity, $message, $file, $line) {
     }
 });
 
-// installed via composer?
-if (file_exists($a = __DIR__.'/../../autoload.php')) {
-    require_once $a;
-} else {
-    require_once __DIR__.'/vendor/autoload.php';
-}
+try {
+    // Maybe this file is used as phar-stub? Let's try!
+    Phar::mapPhar('php-cs-fixer.phar');
+    require_once 'phar://php-cs-fixer.phar/vendor/autoload.php';
+} catch (PharException $e) {
+    // OK, it's not, let give Composer autoloader a try!
+    if (file_exists($a = __DIR__.'/../../autoload.php')) {
+        require_once $a;
+    } else {
+        require_once __DIR__.'/vendor/autoload.php';
+    }
 
```

Comment:
This empty line should be after the closing brace below.

Reply:
done